src/support/provisioning/executeGroovy.ts

```diff
@@ -26,11 +26,15 @@ const serverDefaults = {
 
 export const executeGroovy = function (scriptFile: string, replacements?: { [key: string]: string }, jahiaServer: JahiaServer = serverDefaults): void {
     cy.runProvisioningScript({
-        fileContent: '- executeScript: "' + scriptFile + '"',
-        type: 'application/yaml'
-    }, [{
-        fileName: scriptFile,
-        replacements,
-        type: 'text/plain'
-    }], jahiaServer).then(r => r[0]);
+        script: {
+            fileContent: '- executeScript: "' + scriptFile + '"',
+            type: 'application/yaml'
+        },
+        files: [{
+            fileName: scriptFile,
+            replacements,
+            type: 'text/plain'
+        }],
+        jahiaServer
+    }).then(r => r[0]);
```

Comment:
This `.then(r => r[0]);` might do nothing (the result is not consumed)

Reply:
I think it's to simplify the access to the response data, when using: cy.executeGroovy().then(...) to pass directly the result from the provisioningScript execution that is returning an array.